tests/phpunit/tests/Story_Post_Type.php

```diff
@@ -64,15 +64,6 @@ public static function wpSetUpBeforeClass( $factory ) {
 		);
 	}
 
-	public function setUp() {
```

Comment:
Why is this removed?

Reply:
Because it's unneeded and breaks other tests. There's really no need to do `do_action( 'init' )` in the test setup.